package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "nazareth-events",
+  "version": "1.0.0",
+  "description": "A web app for viewing events in Nazareth",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon ./start.js"
```

Comment:
should have `node ./start.js` as start script (I think) just for when it is on Heroku, it runs the `start` script.
You can add a separate `dev` script